### PR TITLE
Modifier move group issue 

### PR DESCRIPTION
**Pull request by qiaozha**
fix issue #770

### CODE REVIEW

**Review by changlong-liu — APPROVED** [review]

File: src/generate/renders/generated/CliCommands.ts [hunk 1]
```diff
@@ -75,6 +77,8 @@ export class CliCommands extends TemplateBase {
         } else if (this.showCustomFunctionName !== '') {
             this.showCustomFunctionName = '';
         }
+        this.groupClientFactoryName = item['clientFactoryName'];
+        this.importClientFactories.push(item['clientFactoryName']);
```
Comment (lirenhe) [inline src/generate/renders/generated/CliCommands.ts:81]: push this.groupClientFactoryName directly to avoid accessing array again.

**Review by lirenhe — APPROVED** [review]

File: src/generate/renders/generated/CliCommands.ts (hunk 1 above)
Comment (qiaozha) [reply src/generate/renders/generated/CliCommands.ts]: changed.

**Review by fengzhou-msft — APPROVED** [review]
Verified on generated code

**Pull request merged**